onchain/rollups/test/foundry/dapp/helper/genScript.ts

```diff
@@ -70,11 +75,21 @@ function buildScript(inputArray: Input[]): string {
         "        CartesiServerManager.ServerManager/StartSession",
         "}",
         "",
+        "get_version() {",
+        "    ./grpcurl \\",
+        "        -plaintext \\",
+        "        -emit-defaults \\",
+        "        -import-path $GRPC_INTERFACES_DIR \\",
+        "        -proto server-manager.proto \\",
+        "        localhost:5001 \\",
+        "        CartesiServerManager.ServerManager/GetVersion",
+        "}",
+        "",
         "add_input() {",
         "    MSG_SENDER=$1",
         "    PAYLOAD=$2",
         "",
-        "    if [[ -z $INPUT_INDEX ]]",
+        "    if [[ -z ${INPUT_INDEX-} ]]",
```

Comment:
Yes. This is necessary because we are now setting the `-u` flag, which raises an error if you try to get the value of an undefined variable. With `${var-default}`, the flag doesn't raise an error because a default value is specified.

Reply:
I see. All good now